src/main/java/teammates/common/util/StatusMessage.java

```diff
@@ -1,8 +1,14 @@
 package teammates.common.util;
 
+import java.io.Serializable;
+
 import teammates.common.util.Const.StatusMessageColor;
 
-public class StatusMessage {
+/**
+ * The {@code StatusMessage} class encapsulates the text of status message 
+ * and its level of seriousness of the status message (the color of the message).
+ */
+public class StatusMessage implements Serializable {
```

Comment:
Ohh. It is to allow the StatusMessage to be added to the session. If not it will throw NotSerializableException when I try to add it to the session.